docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -900,6 +900,15 @@ As the core components use emotion as a styled engine, the props used by emotion
 - Remove `onRendered` prop.
   Depending on your use case either use a [callback ref](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs) on the child element or an effect hook in the child component.
 
+### OutlinedInput
+
+- Remove the `labelWidth` prop because it's not longer need.
```

Comment:
```suggestion
- Remove the `labelWidth` prop. The `label` prop now servers the same purpose, using CSS layout instead of JavaScript measurement to render the gap in the outlined.
```